### DISCUSSION TITLE
"useRange" hook usage

### DISCUSSION THREAD

**Discussion by JperF** [algolia/instantsearch#5624] [Q&A]
When using  the `useRange` hook and calling `refine` it does not refetch from algolia.
```typescript
const {refine, canRefine } = useRange({
  attribute: 'fieldName'
})
```

This is the only docs (https://www.algolia.com/doc/api-reference/widgets/range-input/react-hooks/#hook-usage)  I have found on how to use the hook 

```typescript

import { useRange } from 'react-instantsearch-hooks-web';

function CustomRangeInput(props) {
  const {
    start,
    range,
    canRefine,
    refine,
    sendEvent,
  } = useRange(props);

  return <>{/* Your JSX */}</>;
}
```

Is there another example else where that is working?

**Comment by Haroenv** [comment]
Here's an example based on the standard ui component: https://codesandbox.io/s/busy-fermat-lx5h0m?file=/src/components/RangeInput.tsx
